src/components/MenuItemWithTopDescription.tsx

```diff
@@ -0,0 +1,19 @@
+import React, {ForwardedRef, forwardRef} from 'react';
+import {View} from 'react-native';
+import MenuItem, {MenuItemProps} from './MenuItem';
```

Comment:
```suggestion
import MenuItem, type {MenuItemProps} from './MenuItem';
```
But I think you will need to split imports